### ISSUE TITLE
Support for multiple kinect for 360° tracking

### ISSUE THREAD

**Issue by jbinard** [simlrh/OSVR-Kinect#5]
Hello,

Do you think it could be possible to add support for multiple kinect to get a full 360° tracking ?

There is an open source project which combine multiple kinects for a better tracking : https://github.com/mvm9289/multikinect

Unfortunatly, it's an old project which has been discontinued and it supports only VRPN.

Here is a video : https://www.youtube.com/watch?v=LKi1nEDeZJc

**Comment by simlrh** [comment]
I wish I'd seen this before I started! It looks like it would take very little work to update it to use OSVR and maybe support Kinect v2.

I don't have the time at the moment, but I'd take a pull request obviously.

**Comment by mrredcon** [comment]
It would be really cool if we could combine v2 Kinects and v1 Kinects, since I've got both sitting in a cabinet...

**Comment by jaintex1** [comment]
yes please. I have 2 kinect v1 and I would love to be able to use both
